src/style.css

```
@@ -1 +1,63 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-size: 12px;
+  box-sizing: border-box;
+  font-style: normal;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.container {
+  max-width: 1200px;
+  min-width: 1024px;
+  min-height: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: auto 50px;
+  height: 40px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  margin-right: 50px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  text-align: center;
+  padding: 19px 0;
+  margin-right: 20px;
+}
+
+.nav__link:last-child {
+  margin: 0;
+}
+
+.nav__link:hover {
+  text-decoration: none;
+  color: #00acdc;
+  border-bottom: 4px solid #00acdc;
+  border-top: 4px solid white;
+}
```

```suggestion
.nav__link:hover {
  color: #00acdc;
}
```
При хавері потрібно міняти лише колір